assets/scaffold/recipes/sous_admin/recipe.yml

```diff
@@ -0,0 +1,102 @@
+name: 'Sous Admin'
+description: 'A recipe that contains modules, themes, and configuration for the Sous administration experience.'
+type: 'Sous'
+
+install:
+  # Install core dependencies.
+  - config
+  - help
+  - field
+  - filter
+  - node
+  - path_alias
+  - system
+  - text
+  - user
+  - views
+  # Install Claro as it is a dependency of Gin, but don't import config.
+  - claro
+  # Install Toolbar as it is a dependency of Gin Toolbar.
+  - toolbar
+  - admin_toolbar
+  - admin_toolbar_search
+  - admin_toolbar_tools
+  # Install Gin theme and base modules that extend it.
+  - gin
+  - gin_toolbar
+  # Install Entity Usuage
+  - entity_usage
+  # Install Field Group and dependencies
+  - field_group
+  # Install Field States UI
+  - field_states_ui
+  # Install Login Destination
+  - login_destination
+  # Install Login History
+  - login_history
+  # Install Media Library Edit
+  - media_library_edit
+  # Install Publication Date
+  - publication_date
+  # Install Save and Edit
+  - save_edit
+  # Install Views AJAX History
+  - views_ajax_history
+  # Install Views Bulk Operations
+  - views_bulk_operations
+
+config:
+  import:
+    admin_toolbar: '*'
+    admin_toolbar_search: '*'
+    admin_toolbar_tools: '*'
+    entity_usage:
+      - entity_usage.settings
+    gin:
+      - gin.settings
+      - block.block.gin_breadcrumbs
+      - block.block.gin_content
+      - block.block.gin_help
+      - block.block.gin_local_actions
+      - block.block.gin_messages
+      - block.block.gin_page_title
+      - block.block.gin_primary_local_tasks
+    gin_login: '*'
```

Comment:
An error gets thrown on recipe install because gin_login isn't one of the installed modules. Not sure if you want to add it or remove it...

Reply:
@laura-johnson I removed this module. It's nice but unnecessary. For recent projects I've noticed we tend to use the FE theme to style the login page which I think fits better.